src/components/ScanQRModal/index.tsx

```diff
@@ -54,7 +55,8 @@ export const ScanQRModal = ({ isOpen, onClose, onScan }: Props): React.ReactElem
     }
 
     if (successData) {
-      onScan(successData)
+      const { address } = parsePrefixedAddress(successData)
```

Comment:
Are we not handling the prefix in the `handleScan` callback we pass to `<ScanQRWrapper />` ? I mean, other components are just reading the QR code fine

Reply:
I've moving it into the wrapper and pushed for you to test, but the parsing now removes the `shortName` in every instance of QR scanning. The parsing of the `shortName` has a semi-reliance on the address input field.